### ISSUE TITLE
[bug] - When navigates to a url which has #, the image is not generated on the report

### ISSUE THREAD

**Issue by ghost** [serenity-js/serenity-js#634]
Hi,

I'm using serenity-js 2.14 and I have to navigate to a URL which has # in its name. (e.g. www.urlexample.com/#/)

This is how I'm navigating to URL: Navigate.to(www.urlexample.com/#/)

I'm also taking pictures at every interaction.

However, when the tests are executed and the reported is generated, the screenshot is not presented at the report.

e.g.

![image](https://user-images.githubusercontent.com/45850178/90516859-fb60cf80-e13a-11ea-921b-4b01abfd5b1d.png)

**Comment by jan-molak** [comment]
Hi @rodrigo-costa-sofist! Would you be able to create a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) using one of the Serenity/JS template projects and publish it to github so that I can try to reproduce it locally?

Thanks!
Jan

**Comment by viper3400** [comment]
Hi @jan-molak 

I've been able to reproduce this:
* Use the serenity-js-jasmine-protractor-template
* Change the configuration of the Photographer to ```Photographer.whoWill(TakePhotosOfInteractions)``` in ```protractor.conf.js```
* Create the following test step (the url is a random url with a hash I found in the web ... ):

```
it(`should navigate to an url with a hash`, () => 
    actorCalled('Jasmine').attemptsTo(
          UseAngular.disableSynchronisation(),
          Navigate.to('https://www.bounteous.com/#sr=g&m=o&cp=or&ct=-tmc&st=(opu%20qspwjefe)&ts=1402322447'),
          Ensure.that(Website.title(), includes('Bounteous: Transforming brand experiences through co-innovation'))
 ));
```

I think the reason is the file name of the screenshot, wich still contains the #, but the report gets in trouble with this. (The file name is still correct in the generated JSON files).

Finally, the HTML report points to a non-existing image in the src attribute.

![Issue634](https://user-images.githubusercontent.com/11006334/102256170-54831900-3f0b-11eb-9a17-9dc6bb52e809.png)

**Comment by jan-molak** [comment]
Nice catch, thanks for a repro @viper3400!

**Comment by jan-molak** [comment]
Fixed in `2.19.9`
